drf_spectacular/openapi.py

```diff
@@ -435,6 +435,8 @@ def _map_model_field(self, model_field, direction):
         elif hasattr(models, 'JSONField') and isinstance(model_field, models.JSONField):
             # fix for DRF==3.11 with django>=3.1 as it is not yet represented in the field_mapping
             return build_basic_type(OpenApiTypes.OBJECT)
+        elif hasattr(models, 'BinaryField') and isinstance(model_field, models.BinaryField):
```

Comment:
`BinaryField` has existed since Django 1.6, so we don't need to check whether it exists.

```suggestion
        elif isinstance(model_field, models.BinaryField):
```

Reply:
This code is safe. Not happened nothing. We do not want get error.

Reply:
It is unnecessary. Django 1.5 has been end-of-life for years - long before this package existed.

The equivalent check for `JSONField` is required as that was added more recently in Django 3.1 and Django 2.2 is still supported.

Reply:
Well. I will fix.